webpack.config.js

```diff
@@ -29,67 +29,7 @@ const glob = require( 'glob' );
 const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
 const TerserPlugin = require( 'terser-webpack-plugin' );
 const WebpackBar = require( 'webpackbar' );
-
-/**
- * WordPress dependencies
- */
-const LibraryExportDefaultPlugin = require( '@wordpress/library-export-default-webpack-plugin' );
-
-/**
- * Given a string, returns a new string with dash separators converted to
- * camel-case equivalent. This is not as aggressive as `_.camelCase` in
- * converting to uppercase, where Lodash will convert letters following
- * numbers.
- *
- * @param {string} string Input dash-delimited string.
- *
- * @return {string} Camel-cased string.
- */
-function camelCaseDash( string ) {
-	return string.replace(
-		/-([a-z])/g,
-		( match, letter ) => letter.toUpperCase()
-	);
-}
-
-const projectPath = ( relativePath ) => {
-	return path.resolve( fs.realpathSync( process.cwd() ), relativePath );
-};
-
-const externalPackages = [
-	'api-fetch',
-	'compose',
-	'dom-ready',
-	'element',
-	'escape-html',
-	'hooks',
-	'i18n',
-	'is-shallow-equal',
-	'url',
-];
-
-const externals = {
-	react: 'React',
-	'react-dom': 'ReactDOM',
-	tinymce: 'tinymce',
-	moment: 'moment',
-	jquery: 'jQuery',
-	lodash: 'lodash',
-	'lodash-es': 'lodash',
-	'js/googlesitekit/api': [ 'googlesitekit', 'api' ],
-};
-
-[
-	...externalPackages,
-].forEach( ( name ) => {
-	externals[ `@wordpress/${ name }` ] = [ 'wp', camelCaseDash( name ) ];
-} );
-
-const externalEntry = {};
-externalPackages.forEach( ( packageName ) => {
-	const name = camelCaseDash( packageName );
-	externalEntry[ name ] = `./node_modules/@wordpress/${ packageName }`;
-} );
+const { ProvidePlugin } = require( 'webpack' );
 
 // This External Libraries will not part of wp object. Most of this is for Polyfill.
 const externalLibrary = {
```

Comment:
These shouldn't need to be built as external scripts anymore. Can we load the polyfills for all/some entrypoints using a webpack plugin perhaps?

Reply:
Yeah, I think so. Let me have a quick look into that 👍